aws/sdk-codegen/src/main/kotlin/software/amazon/smithy/rustsdk/AwsPresigningDecorator.kt

```diff
@@ -354,3 +339,17 @@ class MoveDocumentMembersToQueryParamsTransform(
         }
     }
 }
+
+private fun RustWriter.documentPresignedMethod() {
+    docs(
+        """
+        Creates a presigned request for this operation. The credentials provider from the `config`
```

Comment:
👍🏻 since these are doc links, you can use `crate::` and not worry about crate-name craziness

Reply:
I think in this case, it's actually fine unlinked since we're referring to the function argument names, and rustdoc will already have the types for those linked up. I did notice another problem though: the `config` arg is only on one of these functions.